2022/templates/L4.svg

```diff
@@ -0,0 +1,1273 @@
+         x="19335.68"
+         y="11220.432"
+         id="text1141"><tspan
+           sodipodi:role="line"
+           id="tspan1139"
+           x="19335.68"
+           y="11220.432"
+           style="fill:#000000;fill-opacity:1;stroke-width:0.264583">Reception</tspan></text>
+      <rect
+         style="fill:none;fill-opacity:1;stroke:#000000;stroke-width:50;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:50, 150;stroke-dashoffset:0;stroke-opacity:1"
+         id="rect1143"
+         width="5999.9995"
+         height="5000"
+         x="8005.9258"
+         y="3445.9292" />
+      <text
+         xml:space="preserve"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:263.758px;line-height:1.25;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:center;letter-spacing:0px;word-spacing:0px;writing-mode:lr-tb;text-anchor:middle;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.264583"
+         x="11004.38"
+         y="5879.4102"
+         id="text1147"><tspan
+           sodipodi:role="line"
+           id="tspan1145"
+           x="11004.38"
+           y="5879.4102"
+           style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:263.758px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:center;writing-mode:lr-tb;text-anchor:middle;stroke-width:0.264583">Photo</tspan><tspan
+           sodipodi:role="line"
+           x="11004.38"
+           y="6209.1074"
+           style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:263.758px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:center;writing-mode:lr-tb;text-anchor:middle;stroke-width:0.264583"
+           id="tspan1149">Studio</tspan></text>
+      <text
+         id="text1066"
+         y="14426.091"
+         x="12680.403"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:352.778px;line-height:1.25;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;letter-spacing:0px;word-spacing:0px;writing-mode:lr-tb;text-anchor:start;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.264583"
+         xml:space="preserve"><tspan
+           sodipodi:role="line"
+           id="tspan1068"
+           x="12680.403"
+           y="14426.091">Test Arena</tspan></text>
```

Comment:
Since it's the test arena there isn't really a need to walk around it since they can go inside it. It does however need spacing away from the wall to allow for the arena's feet.